src/node.cc

```diff
@@ -1580,6 +1581,9 @@ static void Kill(const FunctionCallbackInfo<Value>& args) {
 #define NANOS_PER_SEC 1000000000
 
 // Hrtime exposes libuv's uv_hrtime() high-resolution timer.
+
+// This is the legacy version of hrtime before BigInt was introduced in
+// JavaScript.
```

Comment:
Hmm, seeing this we probably want a note in the documentation for `process.hrtime()` that the BigInt equivalent is preferred.